1.10.5/alpine3.10/Dockerfile

```diff
@@ -105,7 +104,7 @@ RUN echo https://github.com/astronomer/ap-airflow/raw/${REPO_BRANCH}/alpine-pack
 	&& apk del .build-deps py3-numpy-dev \
 	&& ln -sf /usr/bin/python3 /usr/bin/python \
 	&& ln -sf /usr/bin/pip3 /usr/bin/pip \
-	&& sed -i -e '\#https://github.com/astronomer/astronomer/#d' /etc/apk/repositories
+	&& sed -i -e '\#https://github.com/astronomer/#d' /etc/apk/repositories
```

Comment:
I still don't know what this change was for in the original PR

Reply:
The previous one didn't actually remove the line, so our registry was left configured, but it shouldn't be as we don't have a guaranteed stable signing key, and if it changes apk would break